src/Engine/RenderBackend/2D/CanvasRenderer.cpp

```
@@ -125,18 +208,111 @@ void CanvasRenderer::preRender(RenderBackendService *rbSrv) {
+    CanvasRenderer::Font2MeshMap::const_iterator it = font2MeshMap.find(fontKey);
+    if (it != font2MeshMap.end()) {
+        textMesh = it->second;
+        if (textMesh == nullptr) {
+            osre_debug(Tag, "Invalid font mesh detected.");
+            return;
+        } else { 
+            getMeshInfo(textMesh, meshInfoArray, info);
+        }
+    }
+
+    const ui32 lastIndex = textMesh->getLastIndex();
```

cppcheck

[warning] Possible null pointer dereference

(nullPointer)